srml/example/src/lib.rs

```diff
@@ -159,10 +159,10 @@ decl_storage! {
 		// If they have a getter (`get(getter_name)`), then your module will come
 		// equipped with `fn getter_name() -> Type` for basic value items or
 		// `fn getter_name(key: KeyType) -> ValueType` for map items.
-		Dummy get(dummy): T::Balance;
+		Dummy get(dummy): T::Balance : genesis = Default::default();
```

Comment:
What do you think if making `Default::default` the default. So if you just specify `genesis`, you get automatically `Default::default()`.